<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainArea</class>
 <widget class="QWidget" name="MainArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>2400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	color:rgb(0,0,0);
	background-color: rgb(255, 255, 0);
	font:&quot;Microsoft YaHei UI&quot;;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="30,2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1,1,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="4">
         <widget class="VibrateRingButton" name="pushButton_fourpage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	border-image: url(:/DetailBottomIcon/image/MineIcon.jpg);
}

QPushButton:disabled
{
	
	border-image: url(:/DetailBottomIcon/image/MineIcon_Clicked.jpg);
	
}


</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="VibrateRingButton" name="pushButton_secondpage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	border-image: url(:/DetailBottomIcon/image/PicChartIcon.jpg);
}

QPushButton:disabled
{
	border-image: url(:/DetailBottomIcon/image/PicChartIcon_Clicked.jpg);
	
}


</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="VibrateRingButton" name="pushButton_thirdpage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	border-image: url(:/DetailBottomIcon/image/FindIcon.jpg);
}

QPushButton:disabled
{
	
	border-image: url(:/DetailBottomIcon/image/FindIcon_Clicked.jpg);
	
}


</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="VibrateRingButton" name="pushButton_firstpage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	border-image: url(:/DetailBottomIcon/image/DetialBottomIcon.jpg);
}

QPushButton:disabled
{
	border-image: url(:/DetailBottomIcon/image/DetialBottomIcon_Clicked.jpg);
	
}


</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="BookkeepButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/DetailBottomIcon/image/AddIcon.jpg);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VibrateRingButton</class>
   <extends>QPushButton</extends>
   <header location="global">BaseClass/vibrateringbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
